src/components/learner-credit-management/data/utils.js

```diff
@@ -551,3 +555,98 @@ export const isLmsBudget = (
   activeIntegrationsLength,
   isUniversalGroup,
 ) => activeIntegrationsLength > 0 && isUniversalGroup;
+
+/**
+ * Determines if the course has already started. Mostly used around text formatting for tense
+ *
+ * This should also help with reducing 'flaky' tests.
```

Comment:
nit: the flaky tests bit was related to the now-removed jitter, right? If so, this line of the comment docstring likely could be removed.